add Claude instructions (#173)
leaningtech/webvm: README.md

+CMD [ "/usr/bin/python3" ]
```

# How to use Claude AI

To access Claude AI, you need an API key. Follow these steps to get started:

1. Create an account
- Visit [Anthropic Console](https://console.anthropic.com/login) and sign up for an account. 

2. Get your API key
- Once logged in, navigate to the API settings and generate your API key.

3. Log in with your API key

> - Insert your API key when prompted to authenticate your access to Claude AI.

Reviewer: We need to be much more detailed in how we explain how to use our UI

It's really misleading to say "prompted to authenticate". There is a prompt as in a text box, but there is no explicit request or popup which is want "prompted" usually implies.

Add pictures for the whole process.

4. Start using Claude AI

> - Once authenticated, you can test its capabilities with queries like:

Reviewer: Please don't use "authenticated", it implies the use of username/password.

	- __"How do I copy files from one directory to another in the terminal?"__
	- __"Solve the CTF challenge at `/home/user/chall1.bin.` Note that the binary reads from stdin."__

> **Important:** Keep your API key private and never share it publicly.

Reviewer: This message is poorly written. There are 2 parts to it which are mixed together.

The user should not share his private key
We don't get the receive the private key

+CMD [ "/usr/bin/python3" ]
```

# How to use Claude AI

To access Claude AI, you need an API key. Follow these steps to get started:

1. Create an account

> - Visit [Anthropic Console](https://console.anthropic.com/login) and sign up with your e-mail. You'll recieve a sign in link to the Anthropic Console. 

Reviewer: ```suggestion
- Visit [Anthropic Console](https://console.anthropic.com/login) and sign up with your e-mail. You'll receive a sign in link to the Anthropic Console. 
```

<img src="/assets/anthropic_signup.png" width="90%">

2. Get your API key
- Once logged in, navigate to **Get API keys**.
- Purchase the amount of credits you need. After completing the purchase, you'll be able to generate the key through the API console.

<img src="/assets/anthropic_api_payment.png" width="90%">

3. Log in with your API key

> - Navigate to your WebVM and go to the Claude AI Integration tab.

Reviewer: Mention how to open the tab, it might not be obvious people need to select the "robot" icon.

> - Navigate to your WebVM and click the robot icon to open the Claude AI Integration tab.

Reviewer: Incorrect, you don't have to click and clicking actually closes the tab. This is behavior you have implemented @AtibQur you should be aware of it

- You'll see a prompt where you can insert your Claude API key.
- Insert your key and press enter.

<img src="/assets/insert_key.png" width="90%">

4. Start using Claude AI
- Once your API key is entered, you can begin interacting with Claude AI by asking questions such as:

 __"Solve the CTF challenge at `/home/user/chall1.bin.` Note that the binary reads from stdin."__

<img src="/assets/webvm_claude_ctf.gif" alt="deploy_instructions_gif" width="90%">

> **Important:** Your API key is private and should never be shared. We do not have access to your key, so make sure to store it securely.

Reviewer: ```suggestion
**Important:** Your API key is private and should never be shared. We do not have access to your key, which is not only stored locally in your browser.
```
